portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** Class representing a comment object*/
+public class Comment {
+  private String content;
+  private Comment parent;
+  private String postTime;
+  private long id;
+
+  public Comment(String content, String postTime, Comment parent, long id) {
```

Comment:
It looks like parent is null every time you use this class, is this just not fully working yet?

Reply:
Not everything concerning the parent was fully implemented when I submitted this PR/ I've also decided that using an id as opposed to a parent comment might be a better way forward.